app/models/form_profile.rb

```diff
@@ -265,7 +265,7 @@ def initialize_contact_information(user)
   def format_for_schema_compatibility(opt)
     if opt[:address][:street2].blank? && (apt = opt[:address][:street].match(APT_REGEX))
       opt[:address][:street2] = apt[1]
-      opt[:address][:street] = opt[:address][:street].gsub(apt[1], '').strip
+      opt[:address][:street] = opt[:address][:street].gsub(/\W?\s+#{apt[1]}/, '').strip
```

Comment:
what's the `\W?` for here?

Reply:
To catch a comma if it exists.
e.g. '49119 Jadon Mills, Apt 123' and '49119 Jadon Mills Apt 123' would both have the same result